vmm/src/vm.rs

```diff
@@ -743,6 +743,8 @@ impl Vm {
                 .unwrap()
                 .balloon_resize(desired_ram_w_balloon)
                 .map_err(Error::MemoryManager)?;
+            let balloon_size = self.config.lock().unwrap().memory.size - desired_ram_w_balloon;
```

Comment:
Instead of calculating this a second time, could you get the balloon size from the `balloon_resize()` function? Make it return a `Result<u64>` instead of `Result<()>`.

Reply:
Updated